app/presenters/nsm/check_answers/application_status_card.rb

```diff
@@ -3,42 +3,124 @@
 module Nsm
   module CheckAnswers
     class ApplicationStatusCard < Base
+      include GovukLinkHelper
+      include GovukVisuallyHiddenHelper
+      include ActionView::Helpers::UrlHelper
       attr_reader :claim
 
+      delegate :status, to: :claim
+
       def initialize(claim)
         @claim = claim
         @group = 'application_status'
         @section = 'application_status'
       end
 
       def row_data
+        response = status == 'submitted' ? nil : '<p>Fake LAA Response</p>'
+        links = join_strings(*edit_links, *appeal_button)
         [
           {
             head_key: 'application_status',
-            text: "<strong class=\"govuk-tag #{I18n.t("nsm.claims.index.status_colour.#{claim.status}")}\">" \
-                  "#{I18n.t("nsm.claims.index.status.#{claim.status}")}</strong>".html_safe
+            text: sanitize(status_text, tags: %w[strong br p])
           },
-          date_actioned_row
+          (if response
+             {
+               head_key: 'laa_response',
+              text: sanitize(response, tags: %w[p]) + sanitize(links, tags: %w[ul li a p])
+             }
+           end)
         ].compact
       end
 
       private
 
-      def date_actioned_row
-        {
-          head_key: date_actioned_head,
-          text: claim.updated_at&.to_fs(:stamp)
-        }
+      def sanitize(value, tags: [])
+        ApplicationController.helpers.sanitize(value, tags:)
       end
 
-      def date_actioned_head
-        case claim.status
-        when 'submitted'
-          'submitted'
-        when 'granted', 'auto_grant', 'part_grant'
-          'assessed'
+      def status_text
+        if status == 'submitted'
+          join_strings(status_tag(status), submitted_data, translate(:awaiting_review), tag.br, claimed_amount)
         else
-          'returned'
+          join_strings(status_tag(status), submitted_data, tag.br, claimed_amount, allowed_amount(status))
+        end
+      end
+
+      def join_strings(*strings)
+        strings.map { |str| str == tag.br ? str : "<p>#{str}</p>" }.join
+      end
+
+      def claimed_amount
+        "#{NumberTo.pounds(total_gross)} claimed"
+      end
+
+      def total_gross
+        @total_gross ||= Nsm::CostSummary::Summary.new(claim).total_gross
+      end
+
+      # TODO: calculate this amount
+      def allowed_amount(status)
+        return '£0.00 allowed' if status == 'rejected'
+        return "#{NumberTo.pounds(total_gross)} allowed" if status == 'granted'
+
+        'Pending Data'
+      end
+
+      def translate(key)
+        I18n.t("nsm.steps.check_answers.groups.#{group}.#{section}.#{key}")
+      end
+
+      def status_tag(_status)
+        "<strong class=\"govuk-tag #{I18n.t("nsm.claims.index.status_colour.#{claim.status}")}\">" \
```

Comment:
Likewise here, this could be:

```ruby
tag.strong(I18n.t("nsm.claims.index.status.#{claim.status}"), 
                   class: "govuk-tag #{I18n.t("nsm.claims.index.status_colour.#{claim.status}")}"
```